examples/image-to-text/run_pipeline.py

```diff
@@ -109,7 +109,7 @@ def main():
         args.prompt = "<image>\nUSER: What's the content of the image?\nASSISTANT:"
     elif args.prompt is None and model_type == "llava_next":
         args.prompt = "[INST] <image>\nWhat is shown in this image? [/INST]"
-        if args.model_name_or_path == "llava-hf/llava-v1.6-vicuna-13b-hf":
+        if args.model_name_or_path in ["llava-hf/llava-v1.6-vicuna-13b-hf", "llava-hf/llava-v1.6-vicuna-13b-hf"]:
```

Comment:
@KimBioInfoStudio the same name `llava-hf/llava-v1.6-vicuna-13b-hf` is repeated twice

Reply:
Thx, fixed